pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -1117,6 +1133,40 @@ message StatusResponse {
     map<string, string> info = 2;
 }
 
+// LinuxIDMapping represents a single user namespace mapping in Linux.
+message LinuxIDMapping {
+   // container_id is the starting id for the mapping inside the container.
+   uint32 container_id = 1;
+   // host_id is the starting id for the mapping on the host.
+   uint32 host_id = 2;
+   // size is the length of the mapping.
+   uint32 size = 3;
+}
```

Comment:
can you have different mappings for each pod ?

Reply:
This is something we are not targeting in first phase. For now, we are only isolating root user between host and containers.Container level isolation we are focussing in next phase.